### ISSUE TITLE
Add magnet link generation in addition to the hash function.

### ISSUE THREAD

**Issue by dessalines** [ludviglundgren/qbittorrent-cli#105]
I'm currently having to use `transmission-show -m` to show the magnet link for a torrent, but since qbt already has the ability to show a hash, it should probably also support generating a magnet link, which is much more useful.

**Comment by ludviglundgren** [comment]
Hi! Can you elaborate on this a bit further? You want to get the manget link of a torrent in your client, like what the Copy -> Magnet menu in the webui?

![image](https://github.com/user-attachments/assets/7a2e8d47-56b2-4d2d-a74e-091b0fead046)

**Comment by dessalines** [comment]
Yes exactly. It should probably be parsing the torrent file, so as to include the download name and trackers. 

But if you can grab that extra info from just the hash, that also works.
